pkg/prometheus/statefulset_test.go

```diff
@@ -1160,53 +1273,187 @@ func TestSidecarsNoCPULimits(t *testing.T) {
 	}
 }
 
-func TestReplicasConfigurationWithSharding(t *testing.T) {
+func TestSidecarsNoCPUResources(t *testing.T) {
 	testConfig := &operator.Config{
 		ReloaderConfig: operator.ReloaderConfig{
-			CPU:    "0",
-			Memory: "50Mi",
-			Image:  "quay.io/coreos/prometheus-config-reloader:latest",
```

Comment:
This was the only place with the usage of prometheus-config-reloader from coreos and not prometheus-operator. I assume this was a mistake?

Reply:
good catch!